### ISSUE TITLE
Sensor data incomplete

### ISSUE THREAD

**Issue by tsothcott** [bdr99/pymazda#15]
Hi! 

First of all - great repo. I've learnt a lot as I'm working on a separate project, but some of your code has come in handy :) 

I've noticed that the sensor data methods aren't complete, namely `generate_edited_text`, `generate_orientation_data_aa` etc.

Do you have these methods complete?

Thanks again!

**Comment by bdr99** [comment]
Hi @tsothcott. Thanks! Glad you're finding it useful.

When I was implementing the sensor data methods, I found that not all of the data was necessary in order to generate valid sensor data. In particular, I chose not to implement the orientation and motion related values, since the logic behind them is very complex and it seemed to work fine without them. I guess the server can't require orientation data and motion data to be present, since some mobile devices might not have these sensors. There are many parts of the sensor data logic that I don't fully understand, but feel free to ask if you have any questions about how I implemented it.

By the way, for `generate_edited_text()`, I chose to leave that one blank as it would only be populated if the user had inputted some text into a text field.
